### PR TITLE
Added environments

### PR DESCRIPTION
**Pull request by JosePedroNobre**
Change env in vs code 

Screenshot 2021-11-08 at 13 08 37

### CODE REVIEW

**Review by Tiago-Rocha — APPROVED** [review]
LGTM

File: lib/src/shops/shop_details_screen.dart [hunk 1]
```diff
@@ -43,7 +43,7 @@ class ShopDetailScreen extends StatelessWidget {
                   child: ListView(
                     children: [
                       Image.network(
-                          "${BedrockService.api}/${BedrockService.apiVersion}/uploads/${shop.images[0].id}/image"),
+                          "${BedrockService.baseURL}/${BedrockService.apiVersion}/uploads/${shop.images[0].id}/image"),
```
Comment (dominiek) [inline lib/src/shops/shop_details_screen.dart:46]: We should probably wrap this in a utility function or component (uploads are standard objects)

**Review by dominiek — APPROVED** [review]
This is good just to be clear I think we also want the ability to switch environments for (admin) users in settings in the app

File: README.md [hunk 2]
```diff
@@ -64,6 +64,38 @@ Before running the app, make sure you have the Bedrock API running whether on lo
 | `BEDROCK_API`         | https://localhost:2300 | Base url of where the API is running |
 | `BEDROCK_API_VERSION` | 1                      | API Version                          |
 
+
+**Run app in dev** 
+`flutter run lib/env/main_dev.dart`
```
Comment (Andrewngabriel) [inline README.md]: This should be `flutter run lib/src/env/main_dev.dart`. I believe the rest are missing the `/src` as well

**Review by Andrewngabriel — APPROVED** [review]

**Pull request merged**